web/packages/shared/components/AwsLaunchButton/AwsLaunchButton.tsx

```diff
@@ -48,7 +48,8 @@ export class AwsLaunchButton extends React.Component<Props> {
 
   render() {
     const { open } = this.state;
-    const { awsRoles, getLaunchUrl, onLaunchUrl } = this.props;
+    const { awsRoles, getLaunchUrl, onLaunchUrl, isAwsIdentityCenterApp } =
+      this.props;
```

Comment:
oh wow, i forgot about class components. I hope we update this to FC soon

Reply:
Right? I was surprised too when I saw that!